pypesto/hierarchical/relative/problem.py

```diff
@@ -66,43 +65,35 @@ def from_petab_amici(
             petab_problem, amici_model, edatas
         )
 
-    def check_edatas(self, edatas: list[amici.ExpData]) -> bool:
```

Comment:
Check edatas was re-defined completely the same for no reason. It is already defined in AmiciInnerProblem.